app/views/passengers/show.haml

```diff
@@ -35,7 +35,7 @@
     = @passenger.rides_expire.strftime('%m/%d/%Y')
 %p
   %b Registering Dispatcher:
-  = @passenger.registerer ? @passenger.registerer.name : 'Unknown'
+  = @passenger&.registerer&.name
```

Comment:
In this case we know that `@passenger` isn't going to be nil, so you don't need the safe operator on that one. While it's true that it won't _hurt_, it's always a good idea to trust the code. That is, if `@passenger` was indeed `nil` we'd want to know, and loudly.

Reply:
Oh I see I think I got mixed up on what object each &. was operating on